presto-yarn-package/src/main/slider/package/scripts/configure.py

```diff
@@ -38,8 +38,9 @@ def set_configuration(component=None):
 
```

Comment:
Regarding commit message, please follow https://chris.beams.io/posts/git-commit/. Maybe `Add resource groups support`?

Reply:
Changed to "Add resource groups support"